### ISSUE TITLE
Watch scripts

### ISSUE THREAD

**Issue by joel-huang** [xavierpuigf/watch_and_help#1]
It seems like the watch phase training scripts are not pushed to the repository. I assume they run the predicate-train script with some custom arguments and hyperparameters, but I'm not sure what they are. Do let me know if you are able to upload them! Thanks lots.

**Comment by xavierpuigf** [comment]
Hi @joel-huang, I just updated the watch scripts. Please let me know if you run into issues!

**Comment by ducnx** [comment]
Why did you set dropout=0 in the watch script? I ran the script and the model is overfitted to the training set.
